docs/cloud/data-security.rst

```diff
@@ -0,0 +1,21 @@
+Securing your data in Qiskit Runtime
+====================================
+
+To ensure that you can securely manage your data when you use Qiskit Runtime, it is important to know exactly what data is stored and encrypted and how you can delete any stored data.
+ 
+
+Protecting your sensitive data in Qiskit Runtime
+------------------------------------------------
+
+The data that you store in IBM Cloud is encrypted at rest by using a randomly generated key.
+
+
+Deleting your data in Qiskit Runtime
+------------------------------------
+
+Deleting a service instance removes all of the content associated with that instance, such as your jobs, results, parameters, and programs. To delete an instance, from the [Instances page](https://cloud.ibm.com/quantum/instances), find the instance you want to remove, click its overflow menu, then click **Delete**. You will be asked to confirm the deletion.
```

Comment:
The `Instance page` link doesn't render correctly for some reason:

<img width="420" alt="image" src="https://user-images.githubusercontent.com/28284230/200693571-909663b6-0d65-4fb1-940c-a4e988b1dcb1.png">

maybe needs a space before the comma?

Reply:
They're in md instead of rst.